Makefile

```diff
@@ -143,14 +143,13 @@ migration/sqlbindata.go: $(GO_BINDATA_BIN) $(wildcard migration/sql-files/*.sql)
 		-nocompress \
 		migration/sql-files
 
-template/bindata.go: $(GO_BINDATA_BIN) $(wildcard template/*.yml)
-	CHE_VERSION=$(CHE_VERSION) JENKINS_VERSION=$(JENKINS_VERSION) TEAM_VERSION=$(TEAM_VERSION) EXPOSCONTROLLER_VERSION=$(EXPOSCONTROLLER_VERSION) go generate template/generate.go
```

Comment:
need to remove all these variables in make file and also corresponding file from project root folder.

Reply:
same as the previous one bc424de